README.adoc

```diff
@@ -84,12 +84,41 @@ spec:
       password:
         secret: app_database_auth
         key: password
+      database:
+        secret: app_database_auth
+        key: dbname
+      port:
+        secret: app_database_auth
+        key: dbport
     labels:
       app.kubernetes.io/name:
         value: example-app
       database-server:
         secret: app_database_auth
         key: server
+
+--------------------------------------------------------------------------------
+
+Example Bitwarden Secrets Manager values for using these entries.
+
+--------------------------------------------------------------------------------
+Secrets Manager Secret
+
+Name: 
+-------------------------------------------------------------------------------
+app_database_auth
+-------------------------------------------------------------------------------
+
+Value:
+--------------------------------------------------------------------------------
+server: productiondb.example.com
+password: <our_really_strong_c0mpl#X_PASSWORD!>
+username: tannerite
+dbname: tracking
+dbport: 12345
```

Comment:
I would consider not adding dbname and dbport just to keep the example shorter.

Reply:
I was intending to show that we can store the secret with a value in Secrets Manager and store it with a different entry name in OCP.